addons/website_event_track/controllers/event_track.py

```diff
@@ -427,6 +427,33 @@ def track_reminder_toggle(self, track_id, set_reminder_on):
 
         return result
 
+    @http.route('/event/track/send_email_reminder', type="jsonrpc", auth="public", website=True)
+    def send_email_reminder(self, track_id, email_to):
+        """ Send email, to email_to if the user is public otherwise to the user email address,
+        with tracks' reminders for external calendars. """
+        template = self.env.ref("website_event_track.mail_template_data_track_reminder", raise_if_not_found=False)
+        if not template:
+            return {'success': False, 'error': 'missing_template'}
+
+        valid_email_to = tools.email_normalize(email_to if request.env.user._is_public() else request.env.user.email)
+        track = request.env['event.track'].sudo().search([('id', '=', track_id), ('is_published', '=', True)])
```

Comment:
We could align display of tracks and this route, and simply check current user can read the track. It means public / portal can get an email only on published tracks, and internal users can do whatever they want.

Hence: can't we remove sudo() and use standard ACLs ? Or do we need the sudo in some browsing in templates, in which case we have to call access check manually ?